### PR TITLE
Add ability to schedule event in seconds

### PR DESCRIPTION
**Pull request by penrods**
The MycroftSkill.schedule_event() method now accepts an integer in addition to
a datetime for the 'when' parameter.  The integer represents the number of
seconds in the future to fire off the event.  E.g.
```python
   self.schedule_event(some_handler, 7)
```
Will invoke some_handler() seven seconds from now.

Also unified language used in event docstrings.

## How to test
Add this to a skill:

```python
   def initialize(self):
       self.schedule_event(self.test, 15)  # should speak 15 seconds after (re)load

   def test(self, message):
       self.speak("Success!")
```

## Contributor license agreement signed?
CLA [ X ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

File: mycroft/skills/core.py [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ def __stop_timeout():
                 self.bus.emit(Message("mycroft.stop.handled",
                                       {"by": "skill:"+str(self.skill_id)}))
             timer.cancel()
-        except:
+        except:  # noqa
```
Comment (forslund) [inline mycroft/skills/core.py]: Maybe use `except Exception:` instead of bare except. This will catch KeyboardInterrupt and other things.

Comment (penrods) [reply mycroft/skills/core.py]: I'm fine with that on this one.  I'll leave it alone in the except on line 1093.

Comment (penrods) [reply mycroft/skills/core.py]: Done!

**Comment by forslund** [comment]
Working nicely. Merging

**Pull request merged**